### PR TITLE
build out sliding panel UI for controls

### PR DESCRIPTION
**Pull request by freeman-lab**

### CODE REVIEW

File: pages/macroalgae.js [hunk 1]
```diff
@@ -76,7 +80,196 @@ const Index = () => {
...
+                  'calc(9 * 100vw / 12 - 24px)',
+                  'calc(8 * 100vw / 12 - 24px)',
+                ],
+                zIndex: 1000,
+                transition: 'transform 0.2s',
+                transform: expanded ? 'translateX(0)' : 'translateX(-100%)',
+              }}
+            >
+              <Box
+                onClick={() => setExpanded((prev) => !prev)}
```
Comment (katamartin) [inline pages/macroalgae.js:112]: Is it possible to use `as='button'` here? Would be nice if we can easily adopt the accessibility features of a button (keyboard interactions, focus state, etc.)

**Review by katamartin — APPROVED** [review]
the actual panel change lgtm! left a small question about the controls button, but feel free to address later

File: pages/macroalgae.js (hunk 1 above)
Comment (freeman-lab) [reply pages/macroalgae.js:112]: ahhh good call, adding that now!

**Comment by freeman-lab** [comment]
addressed the button issue, good catch! merging in

**Pull request merged**